ush/run_verif_global_in_global_workflow.sh

```diff
@@ -38,11 +28,6 @@ if [[ ${end_ymd} == ${VDATE} ]]; then
     cyc2run=${EDATE_GFS: -2}
 fi
 
-if [[ ${cyc2run} != ${cyc} ]]; then
-    echo "Skipping ${METPCASE} for cycle ${cyc}, will be run on cycle ${cyc2run}"
-    exit 0
-fi
-
 end_cyc=${cyc2run}
 verf_step=${STEP_GFS:-24}
 
```

Comment:
There is still `gfs_cyc` reference at line 35.  Since the design is changing such that verif-global will now run every cycle, I think you could just change the if block to

```bash
export fcyc_list="$cyc"
export vhr_list="$cyc"
```

Reply:
It'll need to be the `else` portion of the existing code. metp loops over all the cycles in a day.

Reply:
OK, that's good to know that it will run just once per day and what section of code is needed.  But since `gfs_cyc` won't be defined anymore, the `if` block should still be removed and replaced with just the needed lines to prevent attempting to retrieve an unassigned variable.

Reply:
Apologies.  I was looking at an old commit.

Reply:
Easy to do when looking at PR comments